src/renderer/components/SplittedChatListAndView.js

```diff
@@ -20,28 +20,41 @@ class SplittedChatListAndView extends React.Component {
   constructor (props) {
     super(props)
     this.state = {
-      selectedChat: this.getDefaultChat()
+      selectedChatId: this.getInitialChatId()
     }
 
     this.onChatClick = this.onChatClick.bind(this)
   }
 
   // Returns the chat which will be shown on startup
-  getDefaultChat() {
+  getInitialChatId() {
     const { deltachat } = this.props
-    return deltachat.chats[0]
+    if(deltachat.chats.length == 0) return null
+    return deltachat.chats[0].id
   }
 
-  onChatClick (chat) {
-    console.log('xx', chat)
-    this.setState({ selectedChat: chat })
+  getSelectedChat() {
```

Comment:
This is essentially the same method as `getChats()` in `ChatView`. Why not just move that method here?

Reply:
Essentially I want to revert the commit I made and pass a chat object into `ChatView` instead of id, so `ChatView` doesn't need to do the lookup. The split component takes care of that.

Reply:
We need it for both. It would be better to have a generic deltachat.getChat(id) method?